recipe/bld.bat

```diff
@@ -1,15 +1,34 @@
-if "%ARCH%"=="32" (
-   set ARCH=Win32
-   set COPYSUFFIX=
-) else (
-  set ARCH=x64
-  set COPYSUFFIX=64
-)
+cd source
 
-msbuild source\allinone\allinone.sln /p:Configuration=Release;Platform=%ARCH%
+:: MSYS includes a link.exe that is not MSVC's linker.  Hide it.
+MOVE %LIBRARY_PREFIX%\usr\bin\link.exe %LIBRARY_PREFIX%\usr\bin\link.exe.backup
```

Comment:
Excellent, thanks.  I noticed the moves were failing, but hadn't investigated why.